index.js

```diff
@@ -61,11 +76,16 @@ lint._report = async (options, spinner) => {
 			throw new Error(`Unable to find valid readme for "${options.filename}"`);
 		}
 
+		options.repoURL = options.filename;
 		options.filename = readme;
 	}
 
-	const file = await lint(options);
-	const {messages} = file;
+	const vfiles = await lint(options);
+	const messages = [];
+	vfiles.forEach(vfile => {
+		vfile.path = path.basename(vfile.path);
```

Comment:
Why are you changing the `.path`?

Reply:
Well, similar line is in the master. I think it is used to make the report only shows the base name, instead of the full file name.
https://github.com/sindresorhus/awesome-lint/blob/0aafda85989e32f6e2996ffd5947429cb893331b/index.js#L91